src/Features/Core/Portable/Completion/CompletionTriggerKind.cs

```diff
@@ -25,6 +28,18 @@ public enum CompletionTriggerKind
         /// <summary>
         /// Completion was triggered for snippets only.
         /// </summary>
-        Snippets
+        Snippets,
+
+        /// <summary>
+        /// Completion was trigger by a direct invocation of the completion feature 
+        /// (ctrl-j in Visual Studio).
+        /// </summary>
+        Invoke,
+
+        /// <summary>
+        /// Completion was triggered with a request to commit if a unique item would be selected 
+        /// (ctrl-space in Visual Studio).
+        /// </summary>
+        InvokeAndCommitIfUnique
```

Comment:
The VS commands are called `list members` and `complete word`. I like `complete word` but not `list members`.